### PR TITLE
adeebkhan.json

### PR DESCRIPTION
**Pull request by akpathan2799**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by akpathan2799** [comment]
please check

File: domains/adeebkhan.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Describe the use of this subdomain",
+    "repo": "https://github.com/akpathan2799/akpathan2799.github.io",
+    "owner": {
+        "username": "akpathan2799",
+        "email": "akpathan2799@gmail.com"
+       
+    },
+    "record": {
+        "CNAME": "akpathan2799.github.io"
```
Comment (CuteDog5695) [inline domains/adeebkhan.json:10]: site gives 404 page

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/adeebkhan.json (hunk 1 above)
Comment (akpathan2799) [reply domains/adeebkhan.json:10]: what should i do i am not able to understand it

Comment (akpathan2799) [reply domains/adeebkhan.json:10]: help

Comment (CuteDog5695) [reply domains/adeebkhan.json:10]: website gives a 404 page, aka it doesnt exist

**Review by iostpa — APPROVED** [review]
website accessible now (its hosted on https://akpathan2799.github.io/adeebkhan/)

**Pull request merged**